shared/fs/header/header.js

```diff
@@ -0,0 +1,135 @@
+// @flow
+import * as React from 'react'
+import * as Types from '../../constants/types/fs'
+import {globalStyles, globalColors, globalMargins, isMobile} from '../../styles'
+import {Avatar, Box, ClickableBox, Icon, Text} from '../../common-adapters'
+import HeaderConnector from './header-container'
+
+type FolderHeaderProps = {
+  breadcrumbItems: Array<Types.PathBreadcrumbItem>,
+  dropdownItems: Array<Types.PathBreadcrumbItem>,
+  isTeamPath: boolean,
+  onOpenBreadcrumb: (path: string) => void,
+  onOpenBreadcrumbDropdown: (
+    dropdownItems: Array<Types.PathBreadcrumbItem>,
+    isTeamPath: boolean,
+    onOpenBreadcrumb: (path: string) => void,
+    targetRect: ?ClientRect
+  ) => void,
+}
+
+const FolderHeader = HeaderConnector(
```

Comment:
Typically our pattern for connected components is to export the view layer as a bare dumb component and export the connected component from the container file. This lets you import `FolderHeader` as a dumb component if you wanted to write a story for it alone. Practically, this means removing `HeaderConnector` from this file and exporting the connected component from `header-container.js`. 

Also in this case, since you've got two associated connected components in this folder (good pattern), you could add an `index.js` to export the connected Header/Popup as named exports. Then more auxiliary uses (like stories) can import from the view files directly, while most practical uses (`routes.js` etc.) can import from the directory.